### PR TITLE
chore(build): run all required steps for linking npm package

### PR DESCRIPTION
**Pull request by alexkrolick**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @alexkrolick has signed the CLA. Thanks for the pull request!

File: package.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
         "start": "yarn start:examples",
         "start:dev": "yarn setup; LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev node --max_old_space_size=8192 node_modules/webpack-dev-server/bin/webpack-dev-server.js --config conf/webpack.config.js --mode development",
         "start:examples": "yarn setup; EXAMPLES=true LANGUAGE=en-US REACT=true BABEL_ENV=dev NODE_ENV=dev node --max_old_space_size=8192 node_modules/react-styleguidist/bin/styleguidist.js server --config conf/styleguide.config.js --mode development",
-        "start:npm": "yarn setup; npm-run-all build:prod:npm build:dev:es",
+        "start:npm": "npm-run-all -s clean build:i18n build:prod:npm copy:styles build:dev:es",
```
Comment (jstoffan) [inline package.json]: What specific issue does this address? It looks like `yarn setup` is already doing everything here except `-s` and `copy:styles`.

Comment (alexkrolick) [reply package.json]: `setup` also running the install step, which seems unnecessary for a dev command.

Comment (alexkrolick) [reply package.json]: Rebased

File: package.json [hunk 2]
```diff
@@ -66,7 +66,7 @@
         "start": "yarn start:examples",
         "start:dev": "yarn setup; LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev node --max_old_space_size=8192 node_modules/webpack-dev-server/bin/webpack-dev-server.js --config conf/webpack.config.js --mode development",
         "start:examples": "yarn setup; EXAMPLES=true LANGUAGE=en-US REACT=true BABEL_ENV=dev NODE_ENV=dev node --max_old_space_size=8192 node_modules/react-styleguidist/bin/styleguidist.js server --config conf/styleguide.config.js --mode development",
-        "start:npm": "yarn setup; npm-run-all build:prod:npm build:dev:es",
+        "start:npm": "yarn setup; npm-run-all build:prod:npm copy:styles build:dev:es",
```
[the remark this answers is not in the document]
Comment (alexkrolick) [reply package.json]: No longer required then - will confirm master works when linked

**Pull request closed without merging**